src/components/StackedBlock/StackedBlock.tsx

```diff
@@ -41,10 +40,8 @@ export const StackedBlock = ({
   children,
   ...other
 }: Props) => {
-  const componentClassName = clsx(styles['stacked-block'], className);
```

Comment:
Could you also remove that [empty style block from the css modules file](https://github.com/chanzuckerberg/edu-design-system/blob/998b6168aa3ad73d81606d0efd3236da1fd6f154/src/upcoming-components/StackedBlock/StackedBlock.module.css#L12)?

Reply:
@dierat, thats a good idea, will remove thanks!